crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -436,6 +436,40 @@ class Foo: ...
 reveal_type(Foo.__class__)  # revealed: Literal[type]
 ```
 
+## Module attributes
```

Comment:
Should we also have a test for multi-level attribute assignment, either `mod.Class.attr = ...` or `Class.NestedClass.attr = ...`?